<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bt_Kunde">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kunden</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_Liferanten">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Liferanten</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_Artikel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Artikel</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tab_Main">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>1141</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Verkauf</string>
     </attribute>
     <widget class="QPushButton" name="bt_Angebote">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Angebote</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_Aftrge">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Aftr√§ge</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_Liferschein">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Liferschein</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_Rechnugen">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Rechnugen</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Einkauf</string>
     </attribute>
     <widget class="QPushButton" name="bt_Buchaltung">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Anfragen</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_Beschtelugen">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Beschtelugen</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_Produktion">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Wareneingag</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_Einkauf_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Eingangsrechnug</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Buchhaltung</string>
     </attribute>
     <widget class="QPushButton" name="bt_Lager">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Lager</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Zeiterfassung</string>
     </attribute>
     <widget class="QPushButton" name="bt_Sonstiges">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Sonstiges</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Produktion</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Lager</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Sonstiges</string>
     </attribute>
    </widget>
   </widget>
   <zorder>tab_Main</zorder>
   <zorder>bt_Kunde</zorder>
   <zorder>bt_Liferanten</zorder>
   <zorder>bt_Artikel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVerkauf">
    <property name="title">
     <string>Verkauf</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEinkauf">
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Einkauf</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuBuchhaltung">
    <property name="title">
     <string>Buchhaltung</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuZeiterfassung">
    <property name="title">
     <string>Zeiterfassung</string>
    </property>
   </widget>
   <addaction name="menuVerkauf"/>
   <addaction name="menuEinkauf"/>
   <addaction name="menuBuchhaltung"/>
   <addaction name="menuZeiterfassung"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
